addons/account_payment/tests/test_payment_refund_wizard.py

```diff
@@ -0,0 +1,35 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account_payment.tests.common import AccountPaymentCommon
+from odoo.tests import tagged
+
+
+@tagged('-at_install', 'post_install')
+class TestPaymentRefundWizard(AccountPaymentCommon):
+
+    def setUp(self):
+        super().setUp()
```

Comment:
```suggestion
    def setUpClass(cls):
        super().setUpClass()
```
Unless there's a good reason not to, you should always you `setUpClass` over `setUp`. Difference is that `setUpClass` only gets once for the test suite. `setUp` gets called for each individual test, often leading to database reads/writes that aren't necessary.